lib/Serialization/DeserializeSIL.cpp

```diff
@@ -3773,7 +3773,7 @@ void SILDeserializer::getAllSILFunctions() {
   }
 }
 
-SILVTable *SILDeserializer::readVTable(DeclID VId) {
+SILVTable *SILDeserializer::readVTable(DeclID VId, bool checkSerializedKind) {
```

Comment:
I think this API (and the other APIs that have a similar named parameter) would be clearer if the `checkSerializedKind` parameter was called something like `disallowSerializedForPackage`.